src/utils/actions.js

```diff
@@ -329,6 +329,7 @@ const actions = [
                   .createBookmark(bmkProp)
                   .then((bookmark) => bookmark.getLayout())
                   .then((layout) => layout.qInfo.qId);
+                await app.doSave();
```

Comment:
You should not do a doSave call here. That is quite heavy on the engine. Better to do a `saveObjects`

Reply:
see new commit down below.